phy/__init__.py

```diff
@@ -26,7 +27,8 @@
 
 __author__ = 'Kwik team'
 __email__ = 'cyrille.rossant at gmail.com'
-__version__ = '0.2.0.dev0'
+__version__ = '0.2.0-dev0'
```

Comment:
I think this needs to be reverted: https://www.python.org/dev/peps/pep-0440/#public-version-identifiers